scripts/generate-cities-cache.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Run this with: $ yarn generate-cities-cache
+ */
+
+const fs = require("fs")
+const https = require("https")
+const path = require("path")
+const queryMap = require("./queryMap")
+const cities = require("../data/cityDataSortedByDisplayPreference.json")
+
+const QUERY_NAME = "MapRendererQuery"
+const TTL = 3600 // 1 hour
+
+const MAX_GRAPHQL_INT = 2147483647
+
+const assetsDir = path.resolve(__dirname, "../Pod/Assets/PreHeatedGraphQLCache")
+const queryData = queryMap()[QUERY_NAME]
+
+cities.forEach(city => {
+  const queryParams = {
+    // We don't use the persisted ID here because the queries may not actually have been persisted yet.
```

Comment:
This means we can safely run it _just_ after generating the query map, i.e. before pushing the queries to MP.